napalm_ios/ios.py

```diff
@@ -142,23 +145,66 @@ def is_alive(self):
             'is_alive': self.device.remote_conn.transport.is_active()
         }
 
+    @staticmethod
+    def _create_tmp_file(config):
+        """Write temp file and for use with inline config and SCP."""
+        tmp_dir = os.getcwd()
```

Comment:
@dbarrosop Are you okay with that (from a security perspective)?

It will default to /tmp, but you can overwrite it using an environment variable.

I am probably okay with it...